src/tests/EdFi.DataManagementService.Tests.E2E/Features/SchoolYearReferenceValidation.feature

```diff
@@ -0,0 +1,138 @@
+Feature: School Year Reference Validation
+  Reference validation on School Years
+
+        Background:
+
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+            When a POST request with list of required "descriptors"
```

Comment:
Usually, we would not have a `When` statement in the `Background`. Should only have `Given` and `And` statements.